src/components/ebay-combobox/index.js

```diff
@@ -63,6 +75,22 @@ module.exports = require('marko-widgets').defineComponent({
                 simulateSpacebarClick: true
             });
         }
+
```

Comment:
I feel like this logic should be simplified to:

```js
if (wasToggled) {
  if (isExpanded) {
    this.expander.expand();
  } else {
    this.expander.collapse();
  }
}
```

The first render check should not be needed since `wasToggled` would be true. This means we could remove https://github.com/eBay/ebayui-core/pull/684/files#diff-69c445b9b29be957915e2a875fc5472dR47, the only change we'd have to make is to make sure that `wasExpanded` defaults to false (currently would be undefined). 

https://github.com/eBay/ebayui-core/pull/684/files#diff-69c445b9b29be957915e2a875fc5472dR45
Would become:

```js
const wasExpanded = this.expanded || false;
```

Reply:
Cool. Thanks for the explanation. Fixing.